src/main/java/com/minecolonies/coremod/items/ModItems.java

```diff
@@ -10,6 +10,7 @@
     public static final Item supplyChest  = new ItemSupplyChestDeployer();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make this line start at column 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)